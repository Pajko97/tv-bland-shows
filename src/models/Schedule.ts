import Show from "./Show";

interface Schedule {
    airdate: string,
    airstamp: string,
    airtime: string,
    id: number,
    image: any
    name:  string,
    number: number,
    rating: {average: any}
    runtime: number
    season: number
    show: Show
    summary: null
    type: string,
    url: string
}

export default Schedule