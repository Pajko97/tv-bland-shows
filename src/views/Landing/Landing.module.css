.landing_wrapper {
    padding:2rem 10rem;
}

.landing_wrapper button {
    background-color:red;
    color: white;
    border: none;
    padding:0.4rem 1.2rem;
}


@media only screen and (max-width: 800px) {
    .landing_wrapper {
        padding: 1rem 2rem;
    }
  }