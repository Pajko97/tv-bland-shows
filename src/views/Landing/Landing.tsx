import React from 'react';
import { Link } from 'react-router-dom';
import style from './Landing.module.css'

export default function Landing() {
  return (
    <div className={style.landing_wrapper}>
        <h1>Welcome to my APP</h1>
        <p>To go to list of the latest shows, please click button below</p>
        <Link to="/shows"><button>GO</button></Link>
    </div>
  );
}
