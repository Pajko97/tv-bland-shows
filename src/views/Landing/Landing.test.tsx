import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import API from '../../api/shows-api';
import Schedule from '../../models/Schedule';
import Landing from './Landing';

it('renders landing', async () => {
    let render;
      render = renderer.create(
          <BrowserRouter>
            <Landing/>
          </BrowserRouter>
      ).toJSON();
  
  expect(render).toMatchSnapshot();

})