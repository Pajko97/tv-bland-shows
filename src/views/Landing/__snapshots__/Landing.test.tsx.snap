// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders landing 1`] = `
<div
  className="landing_wrapper"
>
  <h1>
    Welcome to my APP
  </h1>
  <p>
    To go to list of the latest shows, please click button below
  </p>
  <a
    href="/shows"
    onClick={[Function]}
  >
    <button>
      GO
    </button>
  </a>
</div>
`;
